packages/api/src/module/write/shared/application/events-subscription/events-subscription.spec.ts

```diff
@@ -38,7 +39,8 @@ describe('Events subscription', () => {
     subscription = sut.eventsSubscriptions
       .subscription(sut.randomUuid(), {
         start: { from: { globalPosition: 1 } },
-        retry: { retries: 3, backoff: 'LINEAR', delay: 50 },
+        retry: { backoff: 'FIXED', delay: 50, maxBackoff: 1000, resetBackoffAfter: 6 * 1000 },
```

Comment:
Do you have some better name to retry? Maybe something like `eventHandlingRetries`?

Reply:
Too long :wink: